src/OpenTelemetry.Instrumentation.AWS/Implementation/AWSSemanticConventions.cs

```diff
@@ -12,6 +12,9 @@ internal static class AWSSemanticConventions
 
     public const string AttributeAWSDynamoTableName = "aws.table_name";
     public const string AttributeAWSSQSQueueUrl = "aws.queue_url";
+    public const string AttributeAWSSQSQueueName = "aws.sqs.queue_name";
```

Comment:
Do you really need custom attribute here? If it is related to the messaging, you could probably use some of the generic values https://github.com/open-telemetry/semantic-conventions/blob/v1.26.0/docs/messaging/messaging-spans.md#messaging-attributes

Maybe even it is worth to implement more of them?

Reply:
Yes, this attribute name is specific to AWS SQS queues and not applicable to other messaging services.
The name:

* Aligns with the AWS SDK instrumentation in other languages:
For example, the [Python agent](https://github.com/aws-observability/aws-otel-dotnet-instrumentation/blob/74902e85d03c0a8b6e66d27a4646e623acb03e0c/src/AWS.Distro.OpenTelemetry.AutoInstrumentation/AwsAttributeKeys.cs#L36) uses the same attribute name
* Follows the naming convention of ```aws.$SERVICE.$RESOURCE_NAME```
* AWS SDK team is currently working on defining all AWS resource attributes at the semantic convention level

Reply:
Could you please put information in comment which attributes are not part of sem conv?